events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -89,28 +200,104 @@ WCHOI.3 = {
 	}
 	left_portrait = scope:warchief
 
-	# The orcs need their warchief!
-	option = {
-		trigger = {
-			OR = {
-				culture_group = culture_group:orc_group
-				root = scope:warchief
+	option = {	# Stay
+		trigger = { culture_group = culture_group:orc_group }
+		name = WCHOI_3_OPT_A
+
+		add_opinion = {
+			target = scope:warchief
+			opinion = 30
+			modifier = respect_opinion
+		}
+		scope:warchief = {
+			add_opinion = {
+				target = root
+				opinion = 30
+				modifier = grateful_opinion
+			}
+		}
+
+		ai_chance = {
+			base = 100
+			ai_value_modifier = {
+				ai_greed = -1
+			}
+			opinion_modifier = {
+				opinion_target = scope:warchief
+				multiplier = 2
+			}
+			modifier = {
+				factor = 3
+
+				highest_held_title_tier = tier_county
 			}
 		}
-		name = WCHOI_3_OPT_A
 	}
-	# We'll go our own way
-	option = {
-		trigger = {
-			culture_group = culture_group:orc_group
+	option = {	# Leave
+		trigger = { culture_group = culture_group:orc_group }
+
+		add_prestige = {
+			value = major_prestige_gain
+			multiply = root.highest_held_title_tier
+		}
+		dynasty = {
+			add_dynasty_prestige = {
+				value = major_dynasty_prestige_gain
+				multiply = root.highest_held_title_tier
+			}
 		}
```

Comment:
It's so... powerful. Becoming independent is good itself. I probably would pick this option and Swear Fealty back just to get this prestige. 1000 for duchy while monthly prestige is 1.

Reply:
Changed amount of prestige

>  probably would pick this option and Swear Fealty back just to get this prestige

Changed that option too